spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ExampleRequest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring;
+
+import javax.validation.constraints.NotNull;
+
+public final class ExampleRequest {
```

Comment:
Please add Javadoc for public classes and methods.

Reply:
Thank you for review. I fixed it in 30d27f9.